pkg/apis/core/validation/validation_test.go

```diff
@@ -12965,11 +12965,11 @@ func newNodeNameEndpoint(nodeName string) *core.Endpoints {
 func TestEndpointAddressNodeNameUpdateRestrictions(t *testing.T) {
 	oldEndpoint := newNodeNameEndpoint("kubernetes-node-setup-by-backend")
 	updatedEndpoint := newNodeNameEndpoint("kubernetes-changed-nodename")
-	// Check that NodeName cannot be changed during update (if already set)
+	// Check that NodeName can be changed during update
```

Comment:
add a comment: 
This is to accommodate the case where node IP changed or pod CIDR changed.

Reply:
nit:add the comment